pkg/apis/serving/v1alpha2/explainer.go

```diff
@@ -85,3 +92,22 @@ func getExplainer(explainerSpec *ExplainerSpec) (Explainer, error) {
 	}
 	return handlers[0], nil
 }
+
+func getResourceRequirements(explainerSpec *ExplainerSpec) (*v1.ResourceRequirements, error) {
```

Comment:
this should be implemented on AlibiExplainerSpec and you already implemented for CustomSpec

Reply:
What do you mean for this? Do you mean we should add this function in interface and implemented in both AlibiExplainerSpec and CustomSpec?